packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -189,6 +193,11 @@ class WebView extends StatefulWidget {
   ///     * When a navigationDelegate is set HTTP requests do not include the HTTP referer header.
   final NavigationDelegate navigationDelegate;
 
+  /// A function that receives notification when a page has finished loading.
```

Comment:
uber duber nit: a function is called/invoked not receives notifications. I'd replace "A function that receives notification" with "Invoked when"